rhel/SPECS/dbaas-controller.spec

```diff
@@ -0,0 +1,66 @@
+%undefine _missing_build_ids_terminate_build
+%global _dwz_low_mem_die_limit 0
+
+# do not strip debug symbols
+%global debug_package     %{nil}
+
+# the line below is sed'ed by build-server-rpm script to set a correct version
+# https://github.com/Percona-Lab/pmm-submodules/blob/PMM-2.0/build/bin/build-server-rpm
+%global commit            0000000000000000000000000000000000000000
+%global shortcommit       %(c=%{commit}; echo ${c:0:7})
+%define build_timestamp   %(date -u +"%y%m%d%H%M")
+%define release           1
+%define rpm_release       %{release}.%{build_timestamp}.%{shortcommit}%{?dist}
+
+# the line below is sed'ed by build/bin/build-server-rpm to set a correct version
```

Comment:
I think we might group those three sed'ed lines together with a single comment

Reply:
I can group group `full_pmm_version` macro and `commit` variable. Version declaration should be left on it's place.

Reply:
There is no "line below" now

Reply:
Comment removed